runtime/jcl/common/java_lang_Class.cpp

```diff
@@ -1926,7 +1930,9 @@ Java_java_lang_Class_getNestMembersImpl(JNIEnv *env, jobject recv)
 				 * a NoClassDefFoundError
 				 */
 				goto _done;
-			} else if (NULL == nestMember->nestHost) {
+			} else if ((NULL == nestMember->nestHost)
```

Comment:
over here we should add `VM_VMHelpers::currentClass(nestMember);` after popping the object in the special frame